praw/models/reddit/subreddit.py

```diff
@@ -982,9 +1046,28 @@ def submit_video(
             # if thumbnail_path is None, it uploads the PRAW logo
             video_poster_url=self._upload_media(thumbnail_path),
         )
-        return self._submit_media(
-            data, timeout, without_websockets=without_websockets
-        )
+        try:
```

Comment:
I'd suggest DRYing this up, and simply doing a single fetch of the user page. If the a match isn't found, then return `None`.

Reply:
I see. Maybe I'll sleep `timeout` seconds, then fetch.